### PR TITLE
Docs: document contact points for unified alerting

### PR DESCRIPTION
**Pull request by domasx2**
**What this PR does / why we need it**:

Document contact points UI for unified alerting.

This is missing descriptions for individual grafana notification channels. I don't know these well, maybe someone from backend could cointribute?

Related website PR with screenshot: https://github.com/grafana/website/pull/4596

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/alerting/unified-alerting/contact-points.md [hunk 1]
```diff
@@ -1 +1,73 @@
...
+[Webhook](#webhook) | `webhook`
+[Zenduty](#zenduty) | `webhook` 
+
+## Managing contact points for an external alertmanager
+
+Grafana alerting UI supports managing external alertmanager configuration. Add an [alertmanager datasource]({{< relref "../../datasources/alertmanager.md" >}}) and a dropdown will appear at the top of the page, allowing you to select either `Grafana` or an external alertmanager. 
+
+{{< figure max-width="40%" src="/static/img/docs/alerting/unified/contact-points-select-am-8-0.gif" caption="Select alertmanager" >}}
+
+### Editing alertmanager global config
```
Comment (gotjosh) [inline docs/sources/alerting/unified-alerting/contact-points.md:63]:
```suggestion
### Editing global configuration
```

File: docs/sources/alerting/unified-alerting/contact-points.md [hunk 2]
```diff
@@ -1 +1,73 @@
...
+
+To edit global configuration options for an alertmanager, like SMTP server that is used by default for all email contact types:
```
Comment (gotjosh) [inline docs/sources/alerting/unified-alerting/contact-points.md:65]:
```suggestion
Some options can be global for all contact point types. An example of these is the SMTP server that is used by default for all email contact types. These options can be modified by:
```

File: docs/sources/alerting/unified-alerting/contact-points.md [hunk 3]
```diff
@@ -1 +1,73 @@
...
+
+
+1. In the Grafana side bar, hover your cursor over the **Alerting** (bell) icon and then click **Contact points**.
+1. In the dropdown at the top of the page, select a alertmanager datasource.
+1. Click **Edit global config** button at the bottom of the page.
+1. Fill out the form and click **Save global config**.
+
+**Note** this is only for external alertmanagers. Some global options for Grafana contact types, like email settings, can be configured via [Grafana configuration]({{< relref "../../administration/configuration.md" >}}).
```
Comment (gotjosh) [inline docs/sources/alerting/unified-alerting/contact-points.md:73]:
```suggestion
**Note** this is only for external Alertmanagers. Some global options for Grafana contact types, like email settings, can be configured via [Grafana configuration]({{< relref "../../administration/configuration.md" >}}).
```

File: docs/sources/alerting/unified-alerting/contact-points.md [hunk 4]
```diff
@@ -1 +1,73 @@
...
+Sensu | `sensu`
+[Sensu Go](#sensu-go) | `sensugo`
+[Slack](#slack) | `slack`
+Telegram | `telegram`
+Threema | `threema`
+VictorOps | `victorops`
+[Webhook](#webhook) | `webhook`
+[Zenduty](#zenduty) | `webhook` 
+
+## Managing contact points for an external alertmanager
```
Comment (gotjosh) [inline docs/sources/alerting/unified-alerting/contact-points.md:57]:
```suggestion
## Managing contact points for an external Alertmanager
```

File: docs/sources/alerting/unified-alerting/contact-points.md [hunk 5]
```diff
@@ -1 +1,73 @@
...
+
+{{< figure max-width="40%" src="/static/img/docs/alerting/unified/contact-points-select-am-8-0.gif" caption="Select alertmanager" >}}
+
+### Editing alertmanager global config
+
+To edit global configuration options for an alertmanager, like SMTP server that is used by default for all email contact types:
+
+
+1. In the Grafana side bar, hover your cursor over the **Alerting** (bell) icon and then click **Contact points**.
+1. In the dropdown at the top of the page, select a alertmanager datasource.
```
Comment (gotjosh) [inline docs/sources/alerting/unified-alerting/contact-points.md:69]:
```suggestion
1. In the dropdown at the top of the page, select an Alertmanager datasource.
```

**Review by gotjosh — COMMENTED** [review]
Looks Great!

I have a few nits. Ping me and I'll take a look at it again.

**Review by achatterjee-grafana — APPROVED** [review]
Looks good, we can improve in future PRs.

File: docs/sources/alerting/unified-alerting/contact-points.md [hunk 6]
```diff
@@ -1 +1,73 @@
...
+Threema | `threema`
+VictorOps | `victorops`
+[Webhook](#webhook) | `webhook`
+[Zenduty](#zenduty) | `webhook` 
+
+## Managing contact points for an external alertmanager
+
+Grafana alerting UI supports managing external alertmanager configuration. Add an [alertmanager datasource]({{< relref "../../datasources/alertmanager.md" >}}) and a dropdown will appear at the top of the page, allowing you to select either `Grafana` or an external alertmanager. 
+
+{{< figure max-width="40%" src="/static/img/docs/alerting/unified/contact-points-select-am-8-0.gif" caption="Select alertmanager" >}}
```
[the remark this answers is not in the document]
Comment (domasx2) [reply docs/sources/alerting/unified-alerting/contact-points.md:61]: git checkout website repo and run `make docs-local-static`. see https://github.com/grafana/grafana/blob/main/docs/README.md

**Pull request merged**